catchup.sh

```diff
@@ -109,6 +109,9 @@ create_sources() {
    echo "performing Exercise 10 (Create Package with sources)"
    cd create_sources
    conan new hello/0.1 -t -s
+   sed -i -E 's/exports_sources = "src\/\*"/scm = { "type": "git", "url": "auto", "revision": "auto"}/g' conanfile.py
```

Comment:
If the training is being changed to use SCM as default instead of exports_sources, then we should provide the code in the ``create_sources`` folder, with the SCM there. Writing this code is error prone and can make the exercise problematic.

Reply:
I would also recommend to have a slide explaining the ``exports_sources`` alternative (without exercise)